homeassistant/components/switcher_kis/climate.py

```diff
@@ -105,6 +106,9 @@ def __init__(
         self._attr_temperature_unit = UnitOfTemperature.CELSIUS
 
         self._attr_hvac_modes = [HVACMode.OFF]
+        self._attr_supported_features |= (
+            ClimateEntityFeature.TURN_OFF | ClimateEntityFeature.TURN_ON
+        )
```

Comment:
We don't know if ON will be supported at this point?

Reply:
All switcher devices support TURN_ON & TURN_OFF (`DeviceState` parameter)
https://github.com/home-assistant/core/blob/3f7d2da35c1877075078f9322f6ad7d09c8a8658/homeassistant/components/switcher_kis/climate.py#L204-L208

I don't understand why `|` was used here
```suggestion
        self._attr_supported_features = (
            ClimateEntityFeature.TURN_OFF | ClimateEntityFeature.TURN_ON
        )
```